gitman/commands.py

```diff
@@ -66,7 +66,7 @@ def install(*names, root=None, depth=None,
                script errors can be ignored
     - `fetch`: indicates the latest branches should always be fetched
     - `clean`: indicates untracked files should be deleted from dependencies
-
+    - `skip_changes`: indicates uncommited changes should be skipped
```

Comment:
"indicates dependencies with uncommited changes should be skipped"

Reply:
Please use this wording throughout the docs as well.

Reply:
Well, I have adapted the documentation according to your proposals.
 
I think `--force` will override `--skip-changes` because of the if-statement in source.update_files

```python
# Check for uncommitted changes
        if not force:
            log.debug("Confirming there are no uncommitted changes...")
            if skip_changes:
                if git.changes(self.type, include_untracked=clean,
                               display_status=False):
                    msg = ("Skipped update due to uncommitted changes "
                           "in {}").format(os.getcwd())
                    common.show(msg, color='git_changes')
                    return
            else:
                if git.changes(self.type, include_untracked=clean):
                    msg = "Uncommitted changes in {}".format(os.getcwd())
                    raise exceptions.UncommittedChanges(msg)
```

Reply:
We might also considering making `--force` and `--skip-changes` mutually exclusive: https://docs.python.org/3/library/argparse.html#mutual-exclusion